_rules/disable-shortcut-670a30.md

```diff
@@ -0,0 +1,412 @@
+---
+id: 670a30
+name: Printable keys only shortcut can be disabled
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, a mechanism to disable the shortcut exists.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger an action on a [user interface component][].
```

Comment:
Also, @WilcoFiers is pointing out that "ui component" it ambiguous…
https://github.com/act-rules/act-rules.github.io/pull/421#discussion_r390893467